validator/src/main.rs

```diff
@@ -322,6 +335,15 @@ pub fn main() {
                 .value_name("UNIX DOMAIN SOCKET")
                 .help("Stream entries to this unix domain socket path")
         )
+        .arg(
+            Arg::with_name("mnemonic_stdin")
```

Comment:
Maybe `--query-mnemonic` instead? I'm concerned "stdin" might be confused with piping on `stdin`, which is already supported.

Reply:
`--ask-mnemonic` ok?

Reply:
Yep!